actions/installer.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+# SLSA verifier install script
+
+VERIFIER_RELEASE="$1"
```

Comment:
nit: If we stick to bash I'd like us to try and stick to using lowercase local (unexported) variables since it's easier to parse which are exported pre-requisites and which are local. Also, linters like `shellcheck` use this convention when reporting issues.

Reply:
You can install shellcheck in VSCode to get warning about this, I think

Reply:
Done

Reply:
Done